src/txn/txn_ckpt.c

```diff
@@ -1025,9 +1025,9 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
 
     /* Reset the statistics tracked per checkpoint. */
     cache->evict_max_page_size = 0;
-    cache->evict_max_seconds = 0;
-    conn->rec_maximum_hs_wrapup_seconds = 0;
-    conn->rec_maximum_image_build_seconds = 0;
+    cache->evict_max_ms = 0;
+    conn->rec_maximum_hs_wrapup_milliseconds = 0;
+    conn->rec_maximum_image_build_milliseconds = 0;
     conn->rec_maximum_seconds = 0;
```

Comment:
rec_maximum_seconds should we change this to ms as well？

Reply:
I considered adding this to the changes, however, I did check the stats for this one and it was showing up in the metrics that were uploaded on the HELP ticket (although it is showing up as 0 on my test/format metrics I uploaded to the ticket- it may be because the workload is a lot less) so I was not sure if that change was necessary.

Reply:
I think we should.

Reply:
I'll add these to the changes once we have agreed upon which unit to use.

Reply:
Changing everything into milliseconds would allow easy understanding.

Reply:
changed here: [182a7db](https://github.com/wiredtiger/wiredtiger/pull/9055/commits/182a7db1c07ad39d9f55d68611382ff4baedcafc)